projects/plugins/jetpack/extensions/blocks/sharing-button/class-sharing-source-block.php

```diff
@@ -568,7 +568,7 @@ public function get_name() {
 	 */
 	public function process_request( $post, array $post_data ) {
 		$post_id = $post instanceof WP_Post ? $post->ID : 0;
-		$fb_url  = $this->http() . '://www.facebook.com/sharer.php?u=' . rawurlencode( $this->get_share_url( $post_id ) ) . '&t=' . rawurlencode( $this->get_share_title( $post_id ) );
+		$fb_url  = $this->http() . '://www.facebook.com/sharer/sharer.php?u=' . rawurlencode( $this->get_share_url( $post_id ) ) . '&t=' . rawurlencode( $this->get_share_title( $post_id ) );
```

Comment:
Any reason to use `$this->http()` (which always returns `https` in this method) instead of hard-coding it?

```suggestion
		$fb_url  = 'https://www.facebook.com/sharer/sharer.php?u=' . rawurlencode( $this->get_share_url( $post_id ) ) . '&t=' . rawurlencode( $this->get_share_title( $post_id ) );
```